httpdocs/css/webots-cloud.css

```
@@ -6,11 +6,11 @@ body {
   height: 100%;
 }
 body:after{
-  content: "beta version";
+  content: "testing";
   position: fixed;
   width: 260px;
   height: 25px;
-  background: #e00f00;
+  background: purple;
```

```suggestion
  background: #e00f00;
```
To go back to the beta layout